api/src/pcapi/scripts/pro/upload_reimbursement_csv_to_offerer_drive.py

```diff
@@ -107,14 +88,14 @@ def _create_and_get_csv_file(user_id: int, parent_folder_id: str, filename: str)
     return link_to_csv
 
 
-def export_csv_and_send_notfication_emails() -> None:
+def export_csv_and_send_notfication_emails(batch_id: int) -> None:
     for email, infos in OFFERER_INFORMATIONS.items():
```

Comment:
```suggestion
    for email, infos in OFFERER_INFORMATION.items():
```
_information_ est indénombrable en anglais